### ISSUE TITLE
Some preview windows jolt up on display

### ISSUE THREAD

**Issue by TheParadoxBox** [sirjuddington/SLADE#1231]
**SLADE Version**: (3.1.1.12)
**Operating System**: (Windows)

**Issue Details**:
I first started using SLADE on Ubuntu Linux. After reinstalling Windows, I noticed that whenever I tried to preview a texture, the preview window would jump up and render a considerable portion unviewable.

![image](https://user-images.githubusercontent.com/77028175/103706006-0dbc9800-4f7a-11eb-8a08-3c8483f7c0fd.png)

This happens in Graphic mode for viewing graphics and flats in Graphic mode and also when viewing palletes. It works fine when I open them in their own tabs, though.

![image](https://user-images.githubusercontent.com/77028175/103706272-976c6580-4f7a-11eb-9961-cb507c56f755.png)

It's a real shame that this is happening, please fix it!

**Comment by sirjuddington** [comment]
I've seen this before but have no idea what causes it, seems to be some weird wxWidgets/driver issue. Out of curiosity, does the same thing happen in 3.2.0 beta?

**Comment by TheParadoxBox** [comment]
Actually, it seems to have gotten worse. See the issue that PenguinDOOM attached.

**Comment by sirjuddington** [comment]
Do you still have this issue? If so I'm not sure there's much I can do about it, see my closing comment on #1217

**Comment by TheParadoxBox** [comment]
No, I haven't had this issue in a while. Not sure when it was fixed but it's not happening anymore. Thanks!
